### PR TITLE
kbd: 2.6.1 -> 2.6.2

### PR DESCRIPTION
**Pull request by trofi**
While at it added trivial updater.

Changes: https://github.com/legionus/kbd/releases/tag/v2.6.2

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/os-specific/linux/kbd/search-paths.patch [hunk 1]
```diff
@@ -52,10 +52,9 @@ Without this patch, kbd will only look inside
  };
 --- a/src/loadkeys.c
 +++ b/src/loadkeys.c
-@@ -27,6 +27,7 @@
+@@ -27,5 +27,6 @@
  
  static const char *const dirpath1[] = {
- 	"",
 +	"/etc/kbd/" KEYMAPDIR "/**",
```
Comment (davidak) [inline pkgs/os-specific/linux/kbd/search-paths.patch:58]: is this patch still needed? i don't see a difference in this line

https://github.com/legionus/kbd/compare/v2.6.1...v2.6.2#diff-c630bb579468ae349fabd8cd7733165012c1db7bf870061ba55095faeed1bf97R29

**Comment by davidak** [comment]
Result of `nixpkgs-review pr 250173` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>kbd</li>
    <li>kbd.dev (kbd.dev.dev ,kbd.dev.vlock)</li>
    <li>kbd.vlock (kbd.vlock.dev ,kbd.vlock.vlock)</li>
  </ul>
</details>

File: pkgs/os-specific/linux/kbd/search-paths.patch (hunk 1 above)
Comment (trofi) [reply pkgs/os-specific/linux/kbd/search-paths.patch:58]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/os-specific/linux/kbd/search-paths.patch adds `/etc/kbd` search paths. I did not change that fact.

The change only adapts to the context change introduced by https://github.com/legionus/kbd/commit/a770f39f79e6a5fdf8f352d8b26d6a50c6b137ff

Does it make sense?

Comment (davidak) [reply pkgs/os-specific/linux/kbd/search-paths.patch:58]: I have mixed this up, sorry. Now it makes sense.

**Review by davidak — APPROVED** [review]
Change looks good

Build successful

nixosTests successful: keymap kbd-setfont-decompress kbd-update-search-paths-patch

updateScript works

Thanks for the update!

**Pull request merged**